File: Perfectnumber.js
function perfectnumbers(number) {
  if (number <= 1) {
    return false;
  }

  let sum = 1;
  for (let i = 2; i < number; i++) { 
    if (number % i === 0) {
      sum += i;
    }
  }
  return sum; 
}

File: Perfectnumber.java
public class Assignment{
   public static boolean perfectnumbers(int number){


if(number <= 1){
return false;
}
sum = 1;
for(int i = 2;i < num; i++){
if(number % i == 0){
sum += i;
  }
    }

   return sum;
  }

}